### PR TITLE
bob-draw:0.1.0

### PR DESCRIPTION
**Pull request by LucaCiucci**
I am submitting
- [x] a new package

Description: A package to show a graphical representation of text diagrams using [svgbob](https://github.com/ivanceras/svgbob)

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

### CODE REVIEW

**Comment by laurmaedje** [comment]
Could you add a bit more uniqueness to the name, to keep level grounds in case someone else also wants to make an svgbob plugin? (A similar thing happened before with gviz and diagraph.)

**Comment by LucaCiucci** [comment]
Thanks for the feedback. Is something like `bob-draw` acceptable or should I use a more original name?

File: packages/preview/bob-draw/0.1.0/lib.typ [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+/// ```typ
+/// render_bob("<--->")
+/// render_bob(
+///   ```
+///   <--->
+///   `,
+///   width: 25%
+/// )
+/// ```
+#let render_bob(
```
Comment (laurmaedje) [inline packages/preview/bob-draw/0.1.0/lib.typ:39]: I would suggest either using kebab-case here or just dropping the `bob` suffix.

File: packages/preview/bob-draw/0.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+```)
+````
+output:
+
+![basic-example](./basic-example.svg)
+
+
+## Full example
+````typ
+#import "@preview/gviz:0.1.0": *
```
Comment (laurmaedje) [inline packages/preview/bob-draw/0.1.0/README.md:25]: copy paste error?

File: packages/preview/bob-draw/0.1.0/typst.toml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "art",
+    "typst",
+    "wasm",
+    "rust",
+    "svgbob",
+    "bob",
+    "diagram",
+    "graph",
+    "draw",
+]
```
Comment (laurmaedje) [inline packages/preview/bob-draw/0.1.0/typst.toml:23]: could you also add `exclude = ["example.typ", "basic-example.typ", "basic-example.svg"]` (or move the examples into a folder and add `exclude = ["examples"]`)

File: packages/preview/bob-draw/0.1.0/typst.toml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+license = "MIT"
+description = "svgbob for typst, powered by wasm"
+
+homepage = "https://github.com/LucaCiucci/bob-typ"
+repository = "https://github.com/LucaCiucci/bob-typ"
+keywords = [
+    "svg",
+    "ascii",
+    "art",
+    "typst",
```
Comment (laurmaedje) [inline packages/preview/bob-draw/0.1.0/typst.toml:15]: this keyword is pretty obvious 😅

**Review by laurmaedje — COMMENTED** [review]
bob-draw is fine!

**Comment by laurmaedje** [comment]
Thank you!

**Pull request merged**